utils/create-module.js

```diff
@@ -110,22 +100,25 @@ const addImportToSassIndex = (cwd, moduleName) => {
   fs.writeFileSync(indexPath, sortedImports.join('\n'), 'utf8');
 };
 
+const addExportToIndexByType = (cwd, moduleName, moduleType) => {
+  switch (moduleType) {
+    case 'css':
+      addExportToSassIndex(cwd, moduleName);
+      break;
+    case 'react':
+      addExportToReactIndex(cwd, moduleName);
+  }
+};
+
 // Only allow this script to be ran off command line
 if (require.main === module) {
   const cwd = process.cwd();
-  const moduleName = process.argv[2];
-  let wantsCss = false;
-  if (process.argv.length >= 3) {
-    const arg = process.argv[3];
-    if (arg.toLowerCase() === 'css') {
-      wantsCss = true;
-    }
-  }
-  addModuleAsDependency(cwd, moduleName, wantsCss);
-  addExportToIndex(cwd, moduleName);
-  if (wantsCss) {
-    // If we add the import by calling this, it throws error:
-    // error Couldn't find package "@workday/canvas-kit-css-alert@^3.0.0-alpha.2" required by "@workday/canvas-kit-css@3.0.0-alpha.2" on the "npm" registry.
-    addImportToSassIndex(cwd, moduleName);
+  const [_, __, moduleName, moduleType = 'react'] = process.argv;
```

Comment:
I personally don't mind the underscores, but it seems like a couple googles the convention is to just use naked commas so I went with that in mine

Reply:
Nice. Prefer naked commas